acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -0,0 +1,163 @@
+## These test the new trapperkeeper-authorization auth.conf default rules
+## as they're specified in FOSS puppet server.
+## The tests are written to assert the curl request was "not forbidden" rather
+## than expecting something meaningful from the endpoint. We're just trying to
+## test that authorization is allowing & rejecting requests as expected.
+##
+## The testing pattern is to call one of the two curl functions with a path,
+## and then one of the two assertion functions to validate allowed/denied.
+## The assertion functions look for regexes and status codes in the stdout
+## of the previous curl invocation.
+
+test_name 'Default auth.conf rules'
+
+step 'Turn on new auth support' do
+  modify_tk_config(
+    master,
+    File.join(master['puppetserver-confdir'], 'puppetserver.conf'),
+    {'jruby-puppet' => {'use-legacy-auth-conf' => false}})
+end
+
+teardown do
+  modify_tk_config(
+    master,
+    File.join(master['puppetserver-confdir'], 'puppetserver.conf'),
+    {'jruby-puppet' => {'use-legacy-auth-conf' => true}})
+end
+
+def curl_authenticated(path)
+  curl = 'curl '
+  curl += '--cert $(puppet config print hostcert) '
+  curl += '--key $(puppet config print hostprivkey) '
+  curl += '--cacert $(puppet config print localcacert) '
+  curl += "--write-out '\\nSTATUSCODE=%{http_code}\\n' "
+  curl += 'https://$(hostname -f):8140'
+  curl += path
+  on(master, curl)
+end
+
+def curl_unauthenticated(path)
+  curl = 'curl --insecure '
```

Comment:
Could also do this by just having the `--cacert ...` setting and not provide the `--cert` or `--key`.  But seems okay either way for this test since it's not about having the client validate the server.